test/lib/util/Severity.test.ts

```diff
@@ -0,0 +1,178 @@
+import {expect} from 'chai';
+import { ENGINE } from '../../../src/Constants';
+import {CustomEslintEngine} from '../../../src/lib/eslint/CustomEslintEngine';
+import {JavascriptEslintEngine, LWCEslintEngine, TypescriptEslintEngine} from '../../../src/lib/eslint/EslintEngine';
+import {CustomPmdEngine, PmdEngine} from '../../../src/lib/pmd/PmdEngine';
+import {RetireJsEngine} from '../../../src/lib/retire-js/RetireJsEngine';
+
+describe('normalizeSeverity', () => 
```

Comment:
This file should probably be renamed, since it's not actually testing a file called `Severity.ts`. It feels like the contents of this could be distributed to other files with minimal pain.

Reply:
@jfeingold35 How about `SeverityValues.test.ts`? Or `SeverityScenarios.test.ts`? Since all the tests here have a unified goal and there are quite a few of them, I feel having them together in one place keeps them well organized.

Reply:
What if it was `test/lib/services/RuleEngine.severity.test.ts` or something like that? That way, it's clearly associated with the RuleEngine and its subclasses.